pkg/landscaper/controllers/installations/controller.go

```diff
@@ -174,7 +174,7 @@ func (c *Controller) initPrerequisites(ctx context.Context, inst *lsv1alpha1.Ins
 			currOp, "SetupRegistries", err.Error())
 	}
 
-	intBlueprint, err := blueprints.Resolve(ctx, op.ComponentsRegistry(), lsCtx.External.ComponentDescriptorRef(), inst.Spec.Blueprint)
+	intBlueprint, err := blueprints.Resolve(ctx, op.ComponentsRegistry(), lsCtx.External.ComponentDescriptorRef(), inst.Spec.Blueprint, lsCtx.External.RepositoryContext)
```

Comment:
Why do we need to add lsCtx.External.RepositoryContext? This should be also part of lsCtx.External.ComponentDescriptorRef()?